src/Azure/Akka.Streams.Azure.StorageQueue.Tests/QueueSinkSpec.cs

```diff
@@ -29,8 +29,7 @@ public async Task A_QueueSink_should_add_elements_to_the_queue()
             var t = Source.From(messages)
                 //.Select(x => new QueueMessage(x))
                 .ToStorageQueue(Queue, Materializer);
-
-            t.Wait(TimeSpan.FromSeconds(3)).Should().BeTrue();
+            t.Wait();
```

Comment:
This is very dangerous. If the underlying Task never completes, the test will block indefinetly, making the unit test hang/freeze. The timeout was there to make sure that this never happens, if the Task never completes in 3 seconds, then there is a big probability that we have a faulty stream stage.

Reply:
Thanks for your suggestion. Is it possible to enlarge the timespan? As described in the bug report #593, the test may fail when transient errors happen. If waiting until the task finishes is dangerous, I think it may be better to enlarge the timespan.

Reply:
Yes, that is doable. I think a 10 second bump should be enough.